sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md

```diff
@@ -86,7 +86,7 @@ mkdir build && cd build
 cmake \
  -DCMAKE_CXX_COMPILER=clang++ \
  -DTEST_SUITE_SUBDIRS=SYCL \
- -DSYCL_BE="ext_intel_esimd_emulator" \
+ -DONEAPI_DEVICE_SELECTOR="esimd_emulator" \
```

Comment:
Is this right? These config instructions are for the llvm-test-suite.  

As of today, it's configuration uses -DSYCL_BE which it then converts over to SYCL_DEVICE_FILTER .  
https://github.com/intel/llvm-test-suite/blob/intel/SYCL/lit.cfg.py#L221-#L222

It has no support for -DONEAPI_DEVICE_SELECTOR as a configuration argument to the test suite itself.

There is outstanding PR for switching the llvm-test-suite from SYCL_DEVICE_FILTER to ONEAPI_DEVICE_SELECTOR, but I don't believe it is changing that -DSYCL_BE config param, merely updating its interpretation.    @raaiq1

Reply:
You're right, thanks!